reference.md

```diff
@@ -84,22 +87,158 @@ Record objects
 
 * Linked cells are arrays of objects with the short ID and name of the record. Other fields of the linked record are not populated right now.
 
+* Empty cells are returned as `null`.
+
 * Again, formula values are not included right now.
 
+### In request bodies
+
+* In general, values are interpreted the same way they would be if you typed them into a Fieldbook sheet, whether they are strings or numbers.
+
+* If a field has an input type set on it, that will guide the parsing and interpretation of the value. A text field will coerce all values to strings; a currency field will turn numbers into currency values. Again, this is what would happen if you typed the input into the sheet in the UI.
+
+* For a linked field, an object or a list of objects can be provided that reference linked records by id, e.g.: `[{"id": 1}, {"id": 2}]`. If an `id` is given, any other attributes are ignored.
+
+* If an object without an `id` is given for a linked field, a new record is created in the linked sheet with the given attributes.
+
+* A `null` value clears a cell.
+
+* Values for formula fields are ignored.
+
+Input validation
+----------------
+
+Create and update requests perform validation on the request bodies. An HTTP 400 Bad Request response will be returned if:
+
+* Any key in the request body doesn't match a field in the sheet.
+* Any value fails the validation for the input type (if any) of its field (for instance, passing a string to a number field).
+* Any value is missing from a required field.
+* An object or array is given for a non-link field.
+* A value other than an object or array is given for a link field.
+
 Requests
 --------
 
-This version of the API is read-only, so there are really only two calls:
+### Read a sheet (list of records)
+
+```GET https://api.fieldbook.com/v1/:book_id/:sheet_title```
+
+Retrieves a list of all records in the sheet. Example:
+
+```
+$ curl -u $KEY:$SECRET https://api.fieldbook.com/v1/5643be3316c813030039032e/people
+```
+
+Response (HTTP 200 OK):
+
+```
+[
+  {
+    "id":1,
+    "name":"Alice",
+    "age":23
+  },
+  {
+    "id":2,
+    "name":"Bob",
+    "age":38
+  },
+  {
+    "id":3,
+    "name":"Carol",
+    "age":41
+  },
+]
+```
+
+### Read a record
+
+```GET https://api.fieldbook.com/v1/:book_id/:sheet_title/:record_id```
+
+Retrieves a single record. Example:
+
+```
+$ curl -u $KEY:$SECRET https://api.fieldbook.com/v1/5643be3316c813030039032e/people/1
+```
+
+Response (HTTP 200 OK):
+
+```
+{
+  "id":1,
+  "name":"Alice",
+  "age":23
+}
+```
+
+### Create a record
+
+```POST https://api.fieldbook.com/v1/:book_id/:sheet_title <JSON body>```
+
+With a JSON body representing a single record, creates a record in the sheet. Returns the new record. Example:
+
+```
+$ curl -u $KEY:$SECRET -H "Content-Type: application/json" -X POST \
```

Comment:
I think you don't need `-X POST' on this, I believe curl defaults to POST for requests with data (`-d`)